src/transaction/combine.ts

```diff
@@ -1,41 +1,63 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import {ValidationError} from '../common/errors'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import {JsonObject} from 'ripple-binary-codec/dist/types/serialized-type'
+
+/**
+ * The transactions should all be equal EXCEPT for the 'Signers' field. 
+ */
+ function validateAllSignedTransactionsAreEqual(transactions: Array<JsonObject>) {
+  const exampleTransaction = JSON.stringify({...transactions[0], Signers: null})
+  if (transactions.slice(1).some(tx => JSON.stringify({...tx, Signers: null}) !== exampleTransaction)) {
+    throw new ValidationError('txJSON is not the same for all signedTransactions')
+  }
+}
 
 function addressToBigNumber(address) {
   const hex = Buffer.from(decodeAccountID(address)).toString('hex')
   return new BigNumber(hex, 16)
 }
 
+/**
+ * According to the documentation on multi-signatures:
+ * If presented in binary form, the Signers array must be sorted based on 
+ * the numeric value of the signer addresses, with the lowest value first. 
+ * (If submitted as JSON, the submit_multisigned method handles this automatically.)
+ */
 function compareSigners(a, b) {
   return addressToBigNumber(a.Signer.Account).comparedTo(
     addressToBigNumber(b.Signer.Account)
   )
 }
 
+function getATransactionWithAllSigners(transactions: Array<JsonObject>): JsonObject {
```

Comment:
how about `firstTransactionWithAllSigners`

Reply:
I think 'getA' conveys the intent better than 'first' for this one.

Reply:
there's something about the `ATransaction` in camel case that makes this hard to read. Is it "a transaction"? Or an "a-class transaction"? it looks strange to me. what about just `getTransactionWithAllSigners`

Reply:
Yeah I like that, or `getTransactionAllSigners`

Reply:
You generally don't put articles (a, an, the) in method names, unless they're serving some sort of specific purpose